submissions/kt-std/html-seat-booking/css/style.css

```diff
@@ -0,0 +1,1086 @@
+:root {
+    --background-color: #111;
+    --background-color-lighter: #181818;
+    --background-color-hover: rgba(0, 77, 64, .2);
+    --background-hover: rgba(0, 77, 64, .8);
+    --background-color-darker: #141414;
+    --accent-color: rgba(0, 96, 100, .7);
+    --animation-duration: .4s;
+    --checked-bg-color: rgba(0, 96, 100, 1);
+    --checked-font-color: white;
+    --disabled-bg-color: #444;
+    --disabled-font-color: #666;
+    --row-font-size: 12px;
+    --row-font-color: #666;
+    --font-size: 12px;
+    --space-font: 'Megrim', cursive;
+    --font: 'Quicksand', sans-serif;
+    --padding: 5px 5%;
+    --margin: 5%;
+    --shadow: 0 0 15px white;
+    --text-shadow: 0 0 5px white;
+    --button-color: white;
+    --font-color: #aaa;
+    --thin: 300;
+    --bold: 700;
+    --heading-color: white;
+    --mobile-padding: 40px;
+    --seat-color: #222;
+    --border-radius: 2px;
+    --actor-border: 1px solid rgba(0, 96, 100, .8);
+    --actor-bg: rgba(0, 96, 100, .1);
+    --barcode: 'Libre Barcode 39 Extended Text', cursive;
+    --small: 14px;
+    --medium: 16px;
+}
+
+::-moz-selection,
+::selection {
+    color: white;
+    background-color: var(--checked-bg-color);
+}
+
+* {
+    margin: 0;
+    padding: 0;    
+    box-sizing: border-box;
+    font-family: var(--font);
+    outline: none;
+    line-height: 1.2em;
+    color: var(--font-color);
+}
+
+strong,
+b {
+    font-weight: var(--thin);
+}
+
+body {
+    font-size: var(--medium);
+    font-weight: var(--thin);
+    background-color: var(--background-color);
+}
+
+.movie__container {
+    width: 100%;
+    min-height: 100vh;
+    display: grid;
+    grid-template-rows: repeat(3, auto);
+    grid-template-areas: "poster header header"
+        "poster info info"
+        "poster form form";
+}
+
+.poster__container {
+    min-width: 35vw;
+    min-height: 100vh;
+    position: relative;
+    z-index: 1;
+    grid-area: poster;
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: linear-gradient(90deg, transparent 0%, transparent 55%, #111 100%),
+        url(../assets/poster.jpg);    
+}
+
+.poster__container::before{
+    content: 'Alien';
+    position: absolute;
+    left: 50%;
+    bottom: 5%; 
+    line-height: 1;
+    height: auto;
+    display: block;
+    box-sizing: border-box;
+    transform: translate(-50%);
+    color: white;    
+    font-family: var(--space-font);
+    font-weight: var(--thin);
+    font-size: 50px;
+    letter-spacing: 10px;
+    text-shadow: var(--shadow);
+    text-transform: uppercase;
+    animation: textShadow 4s alternate infinite;    
+}
```

Comment:
This text must be in markup and inside headings tag.
I mean if someone changed the text will it be uppercase?

Reply:
Oops, sorry! :see_no_evil:  I've turned everything back :)